Demo/IntegratorAuthenticationProvider.swift

```diff
@@ -21,12 +21,14 @@ import HyperwalletSDK
 
 public struct IntegratorAuthenticationProvider: HyperwalletAuthenticationTokenProvider {
     private var url: String
-    private let user: String = "userName"
-    private let password: String = "password"
+    private let user: String = "selrestuser@1861681"
+    private let password: String = "Password1!"
+    private let baseUrl = "https://qamaster-hyperwallet.aws.paylution.net"
+    private let userToken = "usr-02d381d3-bf7e-43fe-9d00-02e26b28df57"
     private let session: URLSession
```

Comment:
Looks like you forget to remove the test user info

Reply:
yes, my fault, fixed